packages/components/addon/components/hds/table/td.hbs

```diff
@@ -1,3 +1,6 @@
-<td class="hds-table__td hds-typography-body-200 hds-font-weight-regular" ...attributes>
+<td
+  class="hds-table__td hds-typography-body-200 hds-font-weight-regular{{if @align (concat ' hds-text-' @align)}}"
```

Comment:
[issue] let's try not to do string concatenation and logic in the templates, we tend to do it these operations in a backing class

Reply:
also, we should move the `valign` argument here (in the Td` element, instead of the `Table`) which is more specific to where it's used

Reason: this would also allow consumers to have a more granular control of the vertical alignment of cells (what if for any reason two td cells need to have different vertical alignment? with the `valign` applied to the `table` it would not be possible, and would require a breaking change to our component API to achieve it, with the suggested change it's already possible without any extra change or refactoring)

Reply:
This would be supporting bad design by default. Tables should not be vertically aligned by individual cell but rather at the table level. This has been the table best practice for years.

Reply:
"Guidance over control".